ipn/ipnlocal/local.go

```diff
@@ -1557,10 +1561,27 @@ func endpointsEqual(x, y []tailcfg.Endpoint) bool {
 	return true
 }
 
+// SetNotifyCallback sets the function to call when the backend has something to
+// notify the frontend about.
+//
+// It must be called before Start.
```

Comment:
is this old now? looks like it can change at runtime with that locking? and the other repo still uses it as it was before, right?

If this is true, enforce it with a panic if Start has been called?

Reply:
Yeah this is old, will reword

Reply:
reworded